### PR TITLE
feat: add targetContainer option

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#25

### CODE REVIEW

File: src/Visible.js [hunk 1]
```diff
@@ -46,22 +46,27 @@ const SUPPORT_ELEMENTS_BY_CLASSNAME = (() => {
  */
 class Visible extends Component {
 	/**
-	 * @param {HTMLElement|String|jQuery} [element=document] A base element that detects if another element is visible<ko>엘리먼트가 보이는 기준 엘리먼트</ko>
+	 * @param {HTMLElement|String|jQuery} [baseElement=document] A base element that detects if target elements are visible<ko>타겟 엘리먼트들이 보이는 기준 엘리먼트</ko>
 	 * @param {Object} options The option object of the Visible module<ko>Visible 모듈의 옵션 객체</ko>
+	 * @param {String} [options.targetContainer=element] A target container element that contains the target elements. (baseElement >= targetContainer > target elements) <ko>타겟 엘리먼트들을 포함하고 있는 타겟 컨테이너 엘리먼트 (baseElement >= targetContainer > target elements)</ko>
```
Comment (WoodNeck) [inline src/Visible.js:51]: I think `element` should be changed to `baseElement`

File: src/Visible.js [hunk 2]
```diff
@@ -46,22 +46,27 @@ const SUPPORT_ELEMENTS_BY_CLASSNAME = (() => {
...
 	 * @param {String} [options.targetClass="check_visible"] The class name of the element to be checked<ko>보이는지 확인할 엘리먼트의 클래스 이름</ko>
 	 * @param {Number} [options.expandSize=0] The size of the expanded area to be checked whether an element is visible. If this value is less than zero, the size of the area is smaller than that of the base element. <ko>기준 엘리먼트의 경계를 넘어 엘리먼트가 보이는지 확인할 영역의 크기. 값이 0보다 작으면 엘리먼트가 보이는지 확인할 영역의 크기가 기준 엘리먼트보다 작아진다</ko>
 	 */
-	constructor(element, options) {
+	constructor(baseElement, options) {
 		super();
 		this.options = {
+			targetContainer: baseElement,
```
Comment (WoodNeck) [inline src/Visible.js:58]: One of default `baseElement` assign looks unnecessary, see line 63

**Review by WoodNeck — APPROVED** [review]
LGTM

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**